open-mpi/ompi-release
# hwloc 1.9.x updates for v1.8

From open-mpi/ompi#498:

Bring over all hwloc commits after the hwloc v1.9.1 tag (including the fix for the "Lhwloc1" duplicate symbol issue).  

@rhc54 Can you review / test?

Here's all the commits from master:
- open-mpi/ompi@0c502d9 hwloc README-ompi.txt: update for what we
- open-mpi/ompi@29ccbfd hwloc pci: fix bridge depth
- open-mpi/ompi@1905f35 hwloc: bitmap: fix a corner case in hwloc_bitmap_isincluded() with i
- open-mpi/ompi@5c9157c hwloc: core: only update root->complete sets if insert succeeds
- open-mpi/ompi@dec0109 hwloc: groups: add complete sets when inserting distance/pci groups
- open-mpi/ompi@d6e415c hwloc: AIX: Fix PU os_index
- open-mpi/ompi@80140bb hwloc: distances: when we fail to insert an intermediate group, don'
- open-mpi/ompi@29c9915 hwloc: pci: fix SR-IOV VF vendor/device names
- open-mpi/ompi@da164be hwloc: error: point to the FAQ when displaying the big OS error mess
- open-mpi/ompi@3f96e7a hwloc: synthetic: Misc levels are not allowed in the synthetic descr
- open-mpi/ompi@050bb35 hwloc: x86: use Group instead of Misc for unknown x2apic levels
- open-mpi/ompi@379c7b0 hwloc: x86: use ulong for cache sizes, uint won't be enough in the n
- open-mpi/ompi@6caf9ed hwloc: hpux: improve hwloc_hpux_find_ldom() looking for NUMA node
- open-mpi/ompi@fff1bb5 hwloc: core: reorder children in merge_useless_child() as well
- open-mpi/ompi@77978a8 hwloc: core: fix the merging of identical objects in presence of Mis
- open-mpi/ompi@5427b33 hwloc: debug: fix an overzealous assertion about the parent cpuset v
- open-mpi/ompi@9b59d53 hwloc: cpuid-x86: Fix duplicate asm labels in case of heavy inlining
- open-mpi/ompi@86a536c hwloc: x86 and OSF: Don't forget to set NUMA node nodeset
- open-mpi/ompi@db5bc72 hwloc: API: clearly state that os_index isn't unique while logical_i
- open-mpi/ompi@a636790 hwloc: opal/mca/hwloc/hwloc191/hwloc/NEWS update
- open-mpi/ompi@7c96aec hwloc: errors: improve the advice to send hwloc-gather-topology file
- open-mpi/ompi@d5f8c89 hwloc: configure: fix the check for X11/Xutil.h
- open-mpi/ompi@50b035d hwloc: misc.h: Fix hwloc_strncasecmp() with some icc
- open-mpi/ompi@6764413 hwloc: misc.h: Fix hwloc_strncasecmp() build under strict flags on B
- open-mpi/ompi@6b0011f hwloc: v1.9.1 released, doing 1.9.2rc1 now
